common/src/main/java/com/revenuecat/purchases/common/diagnostics/DiagnosticsAnonymizer.kt

```diff
@@ -1,8 +1,24 @@
 package com.revenuecat.purchases.common.diagnostics
 
-class DiagnosticsAnonymizer {
-    fun anonymizeEventIfNeeded(diagnosticsEvent: DiagnosticsEvent): DiagnosticsEvent {
-        // WIP: Anonymize
-        return diagnosticsEvent
+import com.revenuecat.purchases.common.Anonymizer
+
+class DiagnosticsAnonymizer(
+    private val anonymizer: Anonymizer
+) {
+    fun anonymizeEventIfNeeded(telemetryEvent: DiagnosticsEvent): DiagnosticsEvent {
+        return when (telemetryEvent) {
+            is DiagnosticsEvent.Exception -> anonymizeException(telemetryEvent)
+            is DiagnosticsEvent.Metric -> telemetryEvent
```

Comment:
If we end up sending logs, we will need to anonymize more data, but for now, metrics don't really have any place that we could miss to send PII